Package

```diff
@@ -3,7 +3,7 @@
 plugin_name=iec104
 plugin_type=south
 plugin_install_dirname=iec104
-
+iadditional_libs="usr/local/lib:/usr/local/lib/liblib60870*"
```

Comment:
`iadditional_libs `should be `additional_libs`

Reply:
Thanks for picking that up